source/robot_model/include/robot_model/Model.hpp

```diff
@@ -220,6 +256,25 @@ class Model {
    */
   static bool create_urdf_from_string(const std::string& urdf_string, const std::string& desired_path);
 
+  /**
+   * @brief Compute check if the links of the robot are in collision
+   * @param joint_positions containing the joint positions of the robot
+   * @return true if the robot is in collision, false otherwise
```

Comment:
You can document expected exception 
```suggestion
   * @throws robot_model::exceptions::CollisionGeometryException if collision geometry is not initialized
   * @return true if the robot is in collision, false otherwise
```